daemon/network.go

```diff
@@ -262,6 +263,12 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 		driver = c.Config().Daemon.DefaultDriver
 	}
 
+	isSwarmModeManager := daemon.clusterProvider != nil && daemon.clusterProvider.IsManager()
+	withClusterStore := daemon.configStore.ClusterStore != "" // classic
```

Comment:
I'm not sure looking up configStore is correct here, but netController does not seem to contain cluster store info